core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -1182,6 +1182,15 @@ private static void setLocalDictInfo(CarbonTable table, TableInfo tableInfo) {
     }
   }
 
+  /**
+   * Return the format value defined in table properties
+   * @return String as per table properties, null if not defined
+   */
+  public String getFormat() {
+    return getTableInfo().getFactTable().getTableProperties()
+            .get("format");
```

Comment:
it seems this can be moved to the previous line

Reply:
fixed